sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -116,6 +116,17 @@ abstract class JdbcDialect extends Serializable {
     s"SELECT * FROM $table WHERE 1=0"
   }
 
+  /**
+    * The SQL query that should be used to truncate a table. Dialects can override this method to
```

Comment:
Indentation? The following will check Scala style violations for you.
```
$ dev/scalastyle
```

Reply:
Fixed